app/serializers/invite_serializer.rb

```diff
@@ -44,4 +45,12 @@ def include_redemption_count?
   def expired
     object.expired?
   end
+
+  def warnings
+    object.warnings(scope)
```

Comment:
`Invite#warnings` reaches out to alot of associations so we need to ensure that we eagerload the associations upfront in case this serializer is ever used to display a list of invites.

Reply:
This is a good suggestion and something I wanted to do, but slipped my mind. Unfortunately, eagerloading all associations can be difficult, so I'll just add a `show_warnings` option that will be used only when creating or editing an existing invite (and only the current invite is serialized).